ansible_collections/arista/avd/roles/eos_cli_config_gen/README.md

```diff
@@ -1701,17 +1701,29 @@ ip_http_client_source_interfaces:
 
 ```yaml
 management_api_gnmi:
+  provider: < "eos-native" >
+  transport:
+     grpc:
+     - name: < transport_name >
+       # vrf is optional
+       vrf: < vrf_name >
+      # Per the GNMI specification, the default timestamp field of a notification message is set to be
+      # the time at which the value of the underlying data source changes or when the reported event takes place.
+      # In order to facilitate integration in legacy environments oriented around polling style operations,
+      # an option to support overriding the timestamp field to the send-time is available from EOS 4.27.0F.
+      notification_timestamp: < "send-time" | "last-change-time" >
+      ip_access_group: < acl_name >
```

Comment:
Indentation mismatch in lines 1714 and 1715 with respect to 1707 and 1709.

Reply:
fixed